elementary/monitor/alerts/test.py

```diff
@@ -11,54 +11,40 @@
     DbtTestConfigurationSchema,
     TestResultSchema,
 )
-from elementary.utils.json_utils import try_load_json
-from elementary.utils.log import get_logger
-from elementary.utils.time import DATETIME_FORMAT
-
-logger = get_logger(__name__)
-
-TABLE_FIELD = "table"
-COLUMN_FIELD = "column"
-DESCRIPTION_FIELD = "description"
-OWNERS_FIELD = "owners"
-TAGS_FIELD = "tags"
-SUBSCRIBERS_FIELD = "subscribers"
-RESULT_MESSAGE_FIELD = "result_message"
-TEST_PARAMS_FIELD = "test_parameters"
-TEST_QUERY_FIELD = "test_query"
-TEST_RESULTS_SAMPLE_FIELD = "test_results_sample"
-DEFAULT_ALERT_FIELDS = [
-    TABLE_FIELD,
+from elementary.monitor.api.alerts.normalized_alert import (
     COLUMN_FIELD,
     DESCRIPTION_FIELD,
     OWNERS_FIELD,
-    TAGS_FIELD,
-    SUBSCRIBERS_FIELD,
     RESULT_MESSAGE_FIELD,
+    SUBSCRIBERS_FIELD,
+    TABLE_FIELD,
+    TAGS_FIELD,
     TEST_PARAMS_FIELD,
     TEST_QUERY_FIELD,
     TEST_RESULTS_SAMPLE_FIELD,
-]
+)
+from elementary.utils.json_utils import try_load_json
+from elementary.utils.log import get_logger
+from elementary.utils.time import DATETIME_FORMAT
+
+logger = get_logger(__name__)
 
 
 class TestAlert(Alert):
     TABLE_NAME = "alerts"
+    __test__ = False  # Mark for pytest
```

Comment:
This is because the class name starts with `Test`?
If so, maybe explain that in the comment.